GetIntoTeachingApi/Utils/EmailReconciler.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Net.Mail;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+using MoreLinq;
+using System.Linq;
+
+namespace GetIntoTeachingApi.Utils
+{
+	public static class EmailReconciler
+	{
+        private const string EquivalentEmailHostsFile = "./Fixtures/equivalent_email_hosts.yml";
```

Comment:
Is the `Fixtures` directory the place where config is normally kept with .net ? This sounds like a test fixture.

Reply:
Ah yeah I can see how that would be confusing; I'm not sure what the standard is for .net but I'll make a ticket to look at this and update it in a follow up PR